src/main.css

```diff
@@ -2324,3 +2324,14 @@
     background: #333 !important;
   }
 }
+@-moz-document regexp("https?://github.com/sponsors(/.*)?") {
+  .sponsors-hero-headline {
+    color: #040d21 !important;
```

Comment:
CSS vars pls, same below.

And only use important if absolutely necessary.

once youre done run `make` and force push changes.